glean-core/ios/Glean/Net/HttpPingUploader.swift

```diff
@@ -5,12 +5,16 @@
 /// This class represents a ping uploader via HTTP.
 ///
 /// This will typically be invoked by the appropriate scheduling mechanism to upload a ping to the server.
-class HttpPingUploader {
+public class HttpPingUploader {
```

Comment:
Same as above: > Initializer cannot be declared public because its parameter uses an internal type

It's part of the initializer for the configuration

Reply:
Ahh, totally forgot about that...   carry on then!